app/src/ui/banners/accessibilty-settings-banner.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { Octicon } from '../octicons'
+import * as octicons from '../octicons/octicons.generated'
+import { Banner } from './banner'
+import { LinkButton } from '../lib/link-button'
+import { setBoolean } from '../../lib/local-storage'
+
+export const accessibilityBannerDismissed = 'accessibility-banner-dismissed'
```

Comment:
Should we make this `accessibilityBannerShown` instead and automatically set it once we show it the first time? In other words should we require that the user explicitly dismiss the banner or are we fine with having shown it to them once?

Reply:
> user explicitly dismiss the banner

My thinking was explicitly dismiss it so that users don't miss it. Because if a user has no changes when they first open Desktop, they may not realize the implication of the settings on their day to day until they get into making some changes. 

Other notes.. similar pattern we followed for the announcement about cherry-picking/re-ordering/squashing, the user had to dismiss to not see it again.